apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/service/DexService.java

```diff
@@ -937,4 +989,79 @@ public EthDepositsWithOffset getUserFilledOrders(String user, long offset, long
         return dexContractDao.getAllVersionedForAccountOrder(accountId, orderId, 0, ExchangeContractStatus.values().length - 1);
     }
 
+    public List<String> getAllUsers() throws AplException.ExecutiveProcessException {
+        List<String> addresses = new ArrayList<>();
+        int offset = 0;
+        int limit = 100;
+        boolean isContinue = true;
+
+        while (isContinue) {
```

Comment:
isn't it possible to get stuck here? Using loops of type @forewer (while(true), for(;;)) might require a counter which breaks the loop when being exceeded. Make sense?